manifests/s/Sandboxie/Plus/1.12.4/Sandboxie.Plus.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Sandboxie.Plus
+PackageVersion: 1.12.4
+PackageLocale: en-US
+Publisher: http://xanasoft.com/
+PublisherUrl: https://xanasoft.com
+PublisherSupportUrl: https://github.com/sandboxie-plus/Sandboxie/issues
+Author: https://xanasoft.com
+PackageName: Sandboxie-Plus
+PackageUrl: https://github.com/sandboxie-plus/Sandboxie
+License: GPL-3.0
+LicenseUrl: https://raw.githubusercontent.com/sandboxie-plus/Sandboxie/master/LICENSE
+CopyrightUrl: https://raw.githubusercontent.com/sandboxie-plus/Sandboxie/master/LICENSE
+ShortDescription: Sandboxie is a sandbox-based isolation software for 32- and 64-bit Windows NT-based operating systems.
+Description: |-
+  Sandboxie is a sandbox-based isolation software for 32- and 64-bit Windows NT-based operating systems.
+  It was developed by Sophos (which acquired it from Invincea, which acquired it earlier from the original author Ronen Tzur).

```

Comment:
```suggestion
  It is developed by David Xanatos since it became open source, before that it was developed by Sophos (which acquired it from Invincea, which acquired it earlier from the original author Ronen Tzur).
```